src/mobile/multichoice/test/value.test.ts

```diff
@@ -0,0 +1,97 @@
+import { expect, fixture } from "@open-wc/testing";
+import { MobileMultiChoice } from "../index";
+
+describe("MobileMultiChoice", () => {
```

Comment:
@nmanhit 
Please removing item's label expectation from this file.
It should be placing on items.test.ts.